test/e2e/clusterctl_upgrade_test.go

```diff
@@ -55,27 +54,27 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (CAPV 1.10
 				SkipCleanup:                       skipCleanup,
 				MgmtFlavor:                        testSpecificSettingsGetter().FlavorForMode("topology"),
 				PostNamespaceCreated:              testSpecificSettingsGetter().PostNamespaceCreatedFunc,
-				PreUpgrade:                        vsphereframework.LoadImagesFunc(ctx),
```

Comment:
Why could we drop this? Why did we need it before? (if we needed it)

Reply:
We don't need it anymore when running the second management cluster in kind

Reply:
Talked to Christian, makes sense because we use a 2nd kind cluster now and the core CAPI test already loads images